CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ This changelog is according to [Keep a Changelog](http://keepachangelog.com).
 All notable changes to this project will be documented in this file.
 We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwards.
 
+## 1.10.0: January 31st, 2017
+
+## Added
+* Adds prominent words for Dutch.
+* Adds example for testing the relevant words analysis.
```

Comment:
We are using `relevant words` and `prominent words` interchangeably. We should stick to one. I quickly discussed this with @IreneStr and we decided on `prominent words` because we are using that to describe it in the UI. So the terminology to the outside world is consistent.